apps/website-25/src/pages/api/users/me.ts

```diff
@@ -23,13 +24,25 @@ export default makeApiRoute({
     filterByFormula: `{Email} = "${auth.email}"`,
   }))[0];
 
+  const courseNames = user?.courseSitesVisited.split(',') ?? [];
+
+  if (courseNames.length > 1) {
+    throw new Error('Users with multiple courses are not supported yet');
+  }
+
+  const course = (await db.scan(courseTable, {
+    filterByFormula: `{Course} = "${courseNames[0]}"`,
+  }))[0];
```

Comment:
In the interest of not getting too bogged down in onboarding tickets, I am just looking up the course based on the name here. But it could be good to talk about how to handle data access in general, specifically whether to try and move towards a properly normalised database with an access layer for doing joins etc, vs doing ad hoc lookups like this